frontend/src/components/Dashboard/CrossingListPage/DashboardCrossingListItem/DashboardCrossingListItem.js

```diff
@@ -366,7 +366,7 @@ class DashboardCrossingListItem extends React.Component {
       });
   };
 
-  openClicked = () => {
+  toggleOpen = () => {
```

Comment:
I also am not the biggest fan of openClicked etc. for these names, but I feel like toggleOpen etc. implies this would set the state of openSelected to true or false when called.

I like toggle better than clicked here, just wondering if you had any other naming ideas, maybe something along the lines of selectOpen etc.

Reply:
`activateOpenToggle`